ecosystem/readme.md

```diff
@@ -1,14 +1,118 @@
+# Stellar Ecosystem Proposals (SEPs)
 
-Table of Contents:
-- [sep-0001](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md) stellar.toml file
-- [sep-0002](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0002.md) Federation protocol
-- [sep-0003](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0003.md) Compliance protocol
-- [sep-0004](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0004.md) Tx Status Endpoint
-- [sep-0005](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0005.md) Key Derivation Methods for Stellar Accounts
-- [sep-0006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) Anchor/Client interoperability
-- [sep-0007](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0007.md) URI Scheme to facilitate delegated signing
-- [sep-0008](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0008.md) Regulated Assets
-- [sep-0009](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0009.md) Standard KYC / AML fields
-- [sep-0010](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md) Stellar Web Authentication
-- [sep-0011](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0011.md) Txrep: human-readable low-level representation of Stellar transactions
-- [sep-0012](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md) Anchor/Client customer info transfer
+## SEP Track Terms
+* **Informational** — A SEP on the `Informational` track is one that is open to adoption by the
+  ecosystem, but has not been formally standardized by SDF, and is not endorsed by SDF for
+  adoption. Typically a SEP can start as `Informational` to gain traction within the ecosystem
+  before moving to the `Standards` track.
+* **Standard** — A SEP on the `Standards` track is one that aims for formal standardization and
+  endorsement by SDF for adoption. Typically a Standard has a higher bar towards acceptance, and it
+  requires approval by 2 SDF members of the SEP Team.
+
+## SEP Status Terms
+* **Archived** - A SEP that did not head towards a disposition due to a lack of consensus _and_
+  support. Generally open to revival with additional edits.
```

Comment:
Is this also where rejected SEPs go?

Reply:
Yeah decided to just make it all one flat folder where the metadata on the file says the final word. That way we can use say a static site or the like to show off what's in what stage, such as http://eips.ethereum.org/all